requirements.txt

```diff
@@ -1,3 +1,6 @@
 -e git://github.com/phanrahan/magma.git#egg=mflowgen
 -e git://github.com/mflowgen/mflowgen.git#egg=mflowgen
 -e git://github.com/leonardt/fault.git#egg=fault
+-e git://github.com/phanrahan/peak.git#egg=peak
```

Comment:
I wonder if we can directly include `garnet/requirements.txt` here rather than copying. again future work